### ISSUE TITLE
Unable to return value by using GetEnv('xxx')

### ISSUE THREAD

**Issue by sbInnovation** [deployphp/deployer#2120]
Hi all,

Newbie but already happy to use deployer, I have some issues for using this getEnv() function in my deploy.php.
I have read the doc but nothing specified to add a dependance.

I'm using Laravel 7, Docker, Php 7.4, ...

In .env (local) :
`DEP_USER=deployer`

in deploy.php : 

```
namespace Deployer;

require 'recipe/laravel.php';

// Project name
set('application', 'Test_Validation');

// Project repository
set('repository', 'xxxx');

// [Optional] Allocate tty for git clone. Default value is false.
set('git_tty', false); 

// Shared files/dirs between deploys 
//add('shared_files', []);
//add('shared_dirs', []);


// Writable dirs by web server 
add('writable_dirs', ['storage', 'public', 'bootstrap/cache']);
set('allow_anonymous_stats', false);

echo 'DEP_USER : ' . getenv('DEP_USER'); // For testing

....
```

echo returns 

> DEP_USER : 

Any idea ?

Thx for you help

**Comment by antonmedv** [comment]
https://www.php.net/manual/en/function.getenv.php

**Comment by antonmedv** [comment]
It does not support dot files.

**Comment by sbInnovation** [comment]
Thx for tour response, but do u have any idea for customize deploy.php with a DEP_USER var defined somewhere as your doc suggests it ?

**Comment by antonmedv** [comment]
```
DEP_USER=deployer dep deploy
```

Or 

```
exports DEP_USER=deployer
dep deploy

**Comment by sbInnovation** [comment]
Ok, its a case of use by using the command line dep deploy, is it possible to cumulate many args like that ? 
We agree that is related to your doc here : 

`set('user', function () {
    return getenv('DEP_USER');
});`

**Comment by sbInnovation** [comment]
> ```
> DEP_USER=deployer dep deploy
> ```
> 
> Or
> 
> ```
> export DEP_USER=deployer
> dep deploy
> ```

Sorry but this does not work or I don't understand your explanation
`+ export DEP_USER=deployer
+ ~~~~~~
    + CategoryInfo          : ObjectNotFound: (export:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException`

**Comment by antonmedv** [comment]
Use *nix system.

**Comment by sbInnovation** [comment]
I'm using PowerShell on Windows system in order to deploy my project on linux system, for sure...  :s

**Comment by antonmedv** [comment]
I'm not familiar with PowerShell. You can install WSL and try there.
